bedevere/files.py

```diff
@@ -0,0 +1,20 @@
+"""Checks related to files on a pull request."""
```

Comment:
"file paths in a pull request". Might want to also rename the file since nothing related to this file is actually looking inside the files.

Reply:
Good point.  I've renamed `files` to filepaths`.